tests/access_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"kubevirt.io/kubevirt/tests"
 )
 
-var _ = Describe("User Access", func() {
+var _ = Describe("[rfe_id:500, crit:high, vendor:cnv-qe@redhat.com, level:component]User Access", func() {
```

Comment:
I have one request. Can we align with the way which k8s uses to specify tags? See for instance https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/test/e2e/network/networking_perf.go#L170.

The important difference for me is that they use `[Experimental] [Feature:Networking-IPv6] [Slow]` instead of `[Experimental, Feature:Networking-IPv6, Slow]`

Reply:
While this is not directly related to polarion tags, this  might be an interesting  read too: https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#kinds-of-tests

Reply:
And this one: https://github.com/onsi/ginkgo/issues/144

Reply:
done